doc-releases.md

```diff
@@ -1,21 +1,32 @@
 # Documentation Releases
 
-The following list shows the available versions of Knative documentation. Select
-the version that matches your installed version of Knative.
+The available versions of the Knative documentation can be found on the
+[website](https://www.knative.dev).
+You can also view the documentation source files in one of the branches.
 
-## `knative/docs` repositories
+## Documentation website
+
+Use the Knative Documentation website to learn about the product:
+
+- [`https://www.knative.dev`](https://www.knative.dev)
+
+
+## `knative/docs` branches
+
+If want to fix or add content to a past release, you can find the source files
+in the following branches:
```

Comment:
This feels a little strange. Should this heading be combined with the "released versions" section below, or did you mean to add more here?

Reply:
there are released branches and a development branch but really just want to be clear that you should use the website

Reply:
converted it to a list, ptal